v3/src/components/web-view/web-view.tsx

```diff
@@ -26,6 +31,68 @@ export const WebViewComponent = observer(function WebViewComponent({ tile }: ITi
       <div className="codap-web-view-iframe-wrapper">
         <iframe className="codap-web-view-iframe" ref={iframeRef} src={webViewModel.url} />
       </div>
+      <WebViewDropOverlay />
     </div>
   )
 })
+
+// The WebViewDropOverlay broadcasts notifications to plugins as the user drags and drops attributes.
+function WebViewDropOverlay() {
+  const { tile, tileId } = useTileModelContext()
+  const dropId = `web-view-drop-overlay-${tileId}`
+  const { active, setNodeRef } = useDroppable({ id: dropId })
+  const info = active && getDragAttributeInfo(active)
+  const dataSet = info?.dataSet
+  const attributeId = info?.attributeId
+  // Mouse x and y are tracked so we know where the mouse is when a dragged attribute is dropped
+  const mouseX = useRef<number|undefined>()
+  const mouseY = useRef<number|undefined>()
+
+  useDropHandler(dropId, (_active: Active) => {
+    const { dataSet: dropDataSet, attributeId: dropAttributeId } = getDragAttributeInfo(_active) || {}
+    if (dropDataSet && dropAttributeId && mouseX.current != null && mouseY.current != null) {
+      tile?.applyModelChange(() => {}, {
+        notify: dragWithPositionNotification("drop", dropDataSet, dropAttributeId, mouseX.current, mouseY.current),
+        webViewId: tileId
+      })
+    }
+  })
+
+  if (!dataSet || !attributeId) return null
+
+  const handleMouseMove: MouseEventHandler<HTMLDivElement> = event => {
+    // Determine position within interactiveFrame based on mouse and tile locations
+    const { position } = getTileInfo(tileId ?? "")
+    const tileX = position?.left ?? 0
+    // TODO Hardcoded header heights
+    const kDocumentHeaderHeight = 94
+    const kTileHeaderHeight = 25
+    const tileY = (position?.top ?? 0) + kDocumentHeaderHeight + kTileHeaderHeight
```

Comment:
@kswenson @eireland Is there a better way to determine the mouse position relative within a tile? If not, are the constants I use here available elsewhere in the codebase?